### PR TITLE
add jReject v1.1.4 with git auto-update, close #5817

### LINKED ISSUE #5817: [Request] Add jReject
**Library name:** jReject
**Git repository url:**https://github.com/TurnWheel/jReject
**npm package url(optional):** 
**License(s):** MIT + GPLv2
**Official homepage:** http://jreject.turnwheel.com
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/TurnWheel/jReject
**Watch 27
  Star 265
 Fork 88**
@Amomo  could u help me check this PR for #5817 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/jReject/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "browser",
+    "reject",
+    "popup"
+  ],
+  "version": "1.1.4",
+  "author": {
+    "name": "Steven Bower",
+    "url": "https://github.com/BluSyn"
+  },
+  "licenses": [
```
Comment (PeterDaveHello) [inline ajax/libs/jReject/package.json]: https://github.com/cdnjs/cdnjs/issues/5543

**Comment by maruilian11** [comment]
@Amomo i have changed the licenses field, but i'm not sure if it is right.

File: ajax/libs/jReject/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "MIT",
+    "GPLv2"
```
Comment (Amomo) [inline ajax/libs/jReject/package.json]: According to #5543, we couldn't recognize "GPLv2" but "GPL-2.0". (see [SPDX License List](https://spdx.org/licenses/))
Please use `"GPL-2.0"`.

**Comment by Amomo** [comment]
Hi @maruilian11 I think you should use `"GPL-2.0"` for the licenses field.
Other parts are fine. Don't forget to rebase your commit. Thanks! :-D

**Comment by maruilian11** [comment]
@Amomo thank you very much~ could you help me check this PR again? :grin:

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you~ :-D

**Pull request merged**